src/library/Uno.Material/Styles/Controls/v2/RatingControl.xaml

```diff
@@ -0,0 +1,266 @@
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="muxc:RatingControl">
+                    <Grid x:Name="LayoutRoot">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Disabled">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlForegroundDisabledSelected}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Placeholder">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlPlaceholderForeground}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="PointerOverPlaceholder">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlPlaceholderForegroundPointerOver}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="PointerOverUnselected">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlForegroundPointerOverUnselected}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Set">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlForegroundSelected}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="PointerOverSet">
+                                    <VisualState.Setters>
+                                        <Setter Target="ForegroundContentPresenter.Foreground" Value="{ThemeResource RatingControlForegroundSelected}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+
+                        <StackPanel Grid.Row="0"
+                                    Margin="-20,-20,-20,-20"
```

Comment:
can you remove the negative margin + positive nested margin pairs
to see if this weird hack was even needed?

Reply:
this also seem to ruin the vertical alignment of label vs stars

Reply:
weird, these seem to come straight from: https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/RatingControl/RatingControl.xaml

Reply:
@Xiaoy312 When I remove the negative and positive margins, the Label is being cut off, so some part of it doesn't show. I am trying to play around with the margins to see if anything can fix that cut in the Label without having negative margins.

Reply:
in that case, i guess we will just keep it as it...